_episodes/03-running-containers.md

```diff
@@ -222,28 +232,26 @@ test.txt
 ~~~
 {: .output}
 
-So this shows us that we can exit Docker containers for arbitrary lengths of time and then
+So this shows us that we can exit containers for arbitrary lengths of time and then
 return to our working environment inside of them as desired.
 
 >## Clean up a container
 >
->If you want a container to be [cleaned up][docker-docs-run-clean-up] &mdash; that is
+>If you want a container to be [cleaned up][podman-docks-run-clean-up] &mdash; that is
```

Comment:
"[cleaned up][podman-docks-run-clean-up]" is not properly formatted as a link on the web page